app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,69 @@
+class API::V1::PasswordsController < API::APIController
+
+    #
+    # Route /api/v1/passwords/reset_password
+    #
+    def reset_password
+
+        login_or_email = params[:login_or_email]
+
+        #
+        # Try to find user by login
+        #
+        user = User.find_by_login(login_or_email)
+    
+        if user.nil?
+            #
+            # Try to find user by email
+            #
+            user = User.find_by_email(login_or_email)
+        end
+      
+        #
+        # Could not find user by login or email?
+        #
+        if user.nil?
+            message = "Cannot find user or email."
```

Comment:
I think it would be good to also provide a property that encodes the reason for the 403 response.  So the response could be:
```
{ reason: 'user_not_found', message: message }
```
This way if the client doesn't want to use the message directly they could come up with their own message and just match 'user_not_found'.